src/gobimport/converter.py

```diff
@@ -72,6 +72,30 @@ def _is_literal(field):
     return field[0] == "="
 
 
+def _is_object_reference(field: str):
+    """
+    Returns whether the field references an object attribute
```

Comment:
Probably a real-world example could explain why this is used?

Reply:
Check